### PR TITLE
Update of tested certification matrix

### PR DESCRIPTION
**Pull request by sutaakar**
Signed-off-by: Karel Suta <ksuta@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/master/RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -497,9 +497,9 @@ Kogito operator integrates with various other technologies and operators. The te
 | Technology         | Tested version                                                |
 | ------------------ | ------------------------------------------------------------- |
 | Infinispan         | Infinispan operator 2.1.4 (deployed by OLM `2.1.x` channel)   |
-| Kafka              | Strimzi 0.23.0 (deployed by OLM `stable` channel)             |
-| Keycloak           | Keycloak operator 12.0.3 (deployed by OLM `alpha` channel)    |
+| Kafka              | Strimzi 0.24.0 (deployed by OLM `stable` channel)             |
+| Keycloak           | Keycloak operator 14.0.0 (deployed by OLM `alpha` channel)    |
 | Prometheus         | Prometheus operator 0.47.0 (deployed by OLM `beta` channel)   |
-| Grafana            | Grafana operator 3.10.1 (deployed by OLM `alpha` channel)     |
+| Grafana            | Grafana operator 3.10.3 (deployed by OLM `alpha` channel)     |
```
Comment (ricardozanini) [inline README.md:503]: I think we need to add Knative here as well. See: https://issues.redhat.com/browse/KOGITO-5552

**Review by radtriste — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sutaakar) [reply README.md:503]: Added, from my local test it seems that 0.22 and 0.23 don't work properly for some reason. Will keep the tests on 0.20 until we migrate to the newest version.

File: README.md [hunk 2]
```diff
@@ -497,9 +497,10 @@ Kogito operator integrates with various other technologies and operators. The te
...
+| Knative Eventing   | Knative Eventing 0.20.0                                       |
```
Comment (ricardozanini) [inline README.md:504]:
```suggestion
| Knative Eventing   | Knative Eventing 0.22.1                                       |
```
I've tested :D

Not sure if I'm a reliable source or not.. :sweat:

File: README.md (hunk 1 above)
Comment (ricardozanini) [reply README.md:503]: Weird... I have an env with KogitoSource + KogitoOperator on 0.22.. but you're right, maybe there's something different. As soon as we add everything to the Knative side, we can remove this integration from the operator. Let's keep on 0.20.0

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**